src/app/ngmaterial/ngmaterial.module.ts

```diff
@@ -0,0 +1,54 @@
+import { NgModule } from '@angular/core';
```

Comment:
I feel like this is an antipattern.

We saw the other day how importing modules loads a fresh copy of everything they export except services that are provided in root. this is a lot of memory bloat, isn't it?